metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,104 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alert.engine.eventcondition.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
+}
+
+message MetricSelection {
+  string key = 1;
+  string scope = 2;
+  MetricAggregationFunction metric_aggregation_function = 3;
+  string metric_aggregation_interval = 4; //iso duration string PT15S
+  repeated Filter filters = 5;
+}
+
+enum MetricAggregationFunction {
+  METRIC_AGGREGATION_FUNCTION_TYPE_UNSPECIFIED = 0;
+  METRIC_AGGREGATION_FUNCTION_TYPE_SUM = 1;
+  METRIC_AGGREGATION_FUNCTION_TYPE_AVG = 2;
+}
+
+message Filter {
+  oneof filter {
+    LeafFilter leaf_filter = 1;
+    CompositeFilter composite_filter = 2;
+  }
+}
+
+message LeafFilter {
+  LhsExpression lhs_expression = 1;
+  Operator operator = 2;
+  RhsExpression rhs_expression = 3;
+}
+
+enum Operator {
+  OPERATOR_UNSPECIFIED = 0;
+  OPERATOR_EQ = 1;
+  OPERATOR_AND = 2;
+  OPERATOR_OR = 3;
+}
+
+message CompositeFilter {
+  Operator operator = 1;
+  Filter lhs_filter = 2;
+  Filter rhs_filter = 3;
+}
+
+
+message LhsExpression {
+  oneof value {
+    ColumnIdentifier column_identifier = 1;
+  }
+}
+
+message RhsExpression {
+  oneof value {
+    Literal literal = 1;
+  }
+}
+
+message ColumnIdentifier {
+  string id = 1;
+}
+
+message Literal {
+  Value value = 1;
+}
+
+message Value {
+  oneof value {
+    string string = 1;
+  }
+}
+
+enum Severity {
+  SEVERITY_UNSPECIFIED = 0;
+  SEVERITY_CRITICAL = 1;
+  SEVERITY_WARNING = 2;
+}
+
+message ViolationCondition {
+  oneof condition {
+    StaticThreshold static_threshold = 2;
+  }
+}
+
+message StaticThreshold {
```

Comment:
nit: It's a type of `ViolationCondition` and now contains more data than just the threshold

```suggestion
message StaticThresholdCondition {
```